src/cmd/create.rs

```diff
@@ -109,28 +109,21 @@ impl Basic {
                     network: network.unwrap_or_default(),
                     key_type: key_type.unwrap_or_default(),
                 };
-                builder
-                    .key_tag(&tag)
-                    .seed_type(self.seed.to_owned())
-                    .seed_words(seed_words)
+                builder.key_tag(&tag).seed_words(seed_words)
             }
         };
         let wallet = builder.create()?;
 
-        verify::print_result(
-            &wallet,
-            &wallet.decrypt(password.as_bytes()),
-            None,
-            opts.format,
-        )
+        verify::print_result(&wallet, &wallet.decrypt(password.as_bytes()), opts.format)
     }
 }
 
 impl Sharded {
     pub async fn run(&self, opts: Opts) -> Result {
-        let seed_words = match &self.seed {
-            Some(seed_type) => Some(get_seed_words(seed_type)?),
-            None => None,
+        let seed_words = if self.seed {
+            Some(get_seed_words()?)
+        } else {
+            None
         };
```

Comment:
```suggestion
        let seed_words = self.seed.then_some(get_seed_words()?);
```

Reply:
Thanks for the suggestion, my Rust newbie-ness is showing.